app/controllers/request_controller.rb

```diff
@@ -1099,15 +1135,27 @@ def render_new_preview
     message = ""
     if @outgoing_message.contains_email?
       if @user.nil?
-        message += _("<p>You do not need to include your email in the request in order to get a reply, as we will ask for it on the next screen (<a href=\"{{url}}\">details</a>).</p>", :url => (help_privacy_path+"#email_address").html_safe);
+        message += _("<p>You do not need to include your email in the " \
+                     "request in order to get a reply, as we will ask " \
+                     "for it on the next screen (<a href=\"{{url}}\">" \
+                     "details</a>).</p>",
+                     :url => (help_privacy_path+"#email_address").html_safe)
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Surrounding space missing for operator `+`.